api/tests/unit/users/test_unit_users_views.py

```diff
@@ -0,0 +1,84 @@
+import json
+
+import pytest
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from organisations.models import Organisation
+from users.models import FFAdminUser
+
+
+@pytest.mark.django_db
+def test_delete_user():
+    def delete_user(
+        user: FFAdminUser, password: str, delete_orphan_organisations: bool = True
+    ):
+        client = APIClient()
+        client.force_authenticate(user)
+        data = {
+            "current_password": password,
+            "delete_orphan_organisations": delete_orphan_organisations,
+        }
+        url = "/api/v1/auth/users/me/"
+        return client.delete(
+            url, data=json.dumps(data), content_type="application/json"
+        )
+
+    # create a couple of users
+    email1 = "test1@example.com"
+    email2 = "test2@example.com"
+    email3 = "test3@example.com"
+    password = "password"
+    user1 = FFAdminUser.objects.create(email=email1)
```

Comment:
We can replace each of these with `FFAdminUser.objects.create_user(email=email1|2|3, password=password)`

Reply:
Right. Replaced